app/views/users/index.html.erb

```diff
@@ -6,7 +6,7 @@
     <td><%= sort_link(@q, :email) %></td>
     <td><%= sort_link(@q, :is_male, 'Gender') %></td>
     <td><%= sort_link(@q, :birthday) %></td>
```

Comment:
In the specification, the listing shows users age, not birthday. You should probably calculate age based on the birthday. Try not to put that logic into view :)